### ISSUE TITLE
Apply species_name_search to basal wedge data

### ISSUE THREAD

**Issue by GregGuerin** [ternaustralia/ausplotsR#8]
'species_name_search' argument in _get_ausplots_ function filters by matches to species names for point intercept and vouchers but is ignored for basal wedge data (all measured species are returned).

Extend the functionality to basal wedge species filtering.

**Comment by tomsaleeba** [comment]
Do you want to give this one a go @Sammunroe ?

**Comment by Sammunroe** [comment]
Happy to have a go, I need to be in the lab all day Monday, but I can work on it after that.

**Comment by Sammunroe** [comment]
Hi Everyone,

I have now edited the basal.wedge=T argument of get_ausplots() so that if species_name_search is specified, veg.basal will only return the raw data for species matching the search term. In addition, I have edited the get_ausplots() R documentation to more clearly explain how a species_name_search will affect what data is extracted for each data.frame. I think those were the only changes that were requested? 

I have made these changes under a separate branch called "basal_species".  As far as I can tell it is working as it should, but could someone just double check it is working properly? If everyone is happy, I can merge it with the master branch. Thanks :)
